Doc/library/http.server.rst

```diff
@@ -55,6 +55,30 @@ The :class:`HTTPServer` and :class:`ThreadingHTTPServer` must be given
 a *RequestHandlerClass* on instantiation, of which this module
 provides three different variants:
 
+.. class:: HTTPSServer(server_address, RequestHandlerClass, \
+                       bind_and_activate=True, *, certfile, keyfile=None, \
```

Comment:
You should mention that some parameters are forwarded to the superclass like `bind_and_activate` which is not in HTTPServer (why is it in HTTPSServer?)

Reply:
`HTTPServer` class also have this parameter because the method
`__init__(self, server_address, RequestHandlerClass, bind_and_activate=True)`
is inherited from `socketserver.TCPServer`. apparently they forgot to specify it in the documentation, I can fix it, if in your opinion this fix corresponds to the PR topic :-)

example:
```python
>>> inspect.signature(HTTPServer)
<Signature (server_address, RequestHandlerClass, bind_and_activate=True)>
```

Reply:
Let's do that in a separate PR I'd say.